Awaymissions are now a memory of the past (almost)

PestoVerde322 opened the pull request:
## About The Pull Request

This PR just removes the Awaymission maps and content, since they're unused.

## Why It's Good For The Game

Faster load time, Less headaches for pathchanges and mappers.

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>

I.O.U. a screenshot, but it all works 👌 

</details>

## Changelog
:cl:
del: Removed the awaymission content and maps
/:cl:

Commit:
Part one of many, the code of the past is gone

Commit:
Part two of possibly many, fixed some stuff and the maps are gone

Commit:
snowdin shuttles begone

PowerfulBacon commented on code/game/objects/effects/spawners/lootdrop.dm:
```diff
@@ -542,3 +542,74 @@
 
+///Former Snowdin loot tables
+
+/obj/effect/spawner/lootdrop/snowdin
```
Do we use these?

PestoVerde322 replied:
yep, in some random ruins in explo!

Rukofamicom commented on code/modules/awaymissions/mission_code/TheFactory.dm:
```diff
@@ -1,835 +0,0 @@
-	speak_chance = 5
-	speak = list("Come get me!","You can't get away!","Die!")
-
-/mob/living/simple_animal/hostile/syndicate/factory/sniper
```
This is the sniper

Rukofamicom commented on code/modules/awaymissions/mission_code/TheFactory.dm:
```diff
@@ -1,835 +0,0 @@
-	visible_message("<span class='boldwarning'>The device activates!</span>")
-	..()
-
-/mob/living/simple_animal/hostile/syndicate/factory/heavy
```
This is the heavy gunner.

Rukofamicom commented on code/modules/mob/living/simple_animal/corpse.dm:
```diff
@@ -272,58 +250,7 @@
-	gloves = /obj/item/clothing/gloves/combat
-	mask = /obj/item/clothing/mask/gas/sechailer/swat/emagged
-	suit = /obj/item/clothing/suit/armor/heavy
-	back = /obj/item/minigunpack
```
And this is the minigun backpack the heavy gunner should not spawn with, and should still be removed. 

It's broken shitcode I fixed up once, but it really doesn't need to exist for players to use.

Rukofamicom reviewed (commented):
Pointing out the two mobs I would like to cherry pick for admin spawning reasons.

Rukofamicom reviewed (approved):
Even if you opt not to keep those, we can always bring them back later ourselves if we really want to. 

When conflicts are fixed up this is a good cleanup.

Commit:
merge conflict?? On deleted files??

How queer!! Ive never seen such a thing- i must inquire about this further with my supervisor post-haste!!

Commit:
i guess we revertin some culled code now

Commit:
backup_nosound moment

Commit:
mergefix

Commit:
Mergefix

spockye reviewed (approved):
checked off map-wise

Commit:
Mergefix 2

Commit:
bnnuy
